Source/Charts/Components/YAxis.swift

```diff
@@ -87,6 +87,12 @@ open class YAxis: AxisBase
     /// **default**: CGFloat.infinity
     @objc open var maxWidth = CGFloat(CGFloat.infinity)
     
+    /// indicates if there should be a fill drawn between 2 limit lines
+    @objc open var limitLineFill = false
+    
+    /// Color of the limit line fill
+    @objc open var limitLineFillColor: NSUIColor? = NSUIColor.green.withAlphaComponent(0.6)
```

Comment:
Where does this colour come from? There are colour templates in the library, please choose an appropriate colour from there.

Reply:
Done, default color now uses one from templates plus alpha

Reply:
Done, default color now uses one from templates plus alpha